README.md

```diff
@@ -551,6 +549,8 @@ extend (whether to other areas of Astronomy or in other domains).
 Release History
 ---------------
 
+4.9.1: 01 August 2017 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.825839.svg)](https://doi.org/10.5281/zenodo.825839)
```

Comment:
Hmm, the one at the top is a pointer to the latestdoi, isn't it? So it will point to the right version when we publish it. Or am I missing something?